src/components/integration/IntegrationSettings.tsx

```
@@ -308,6 +341,73 @@ export const IntegrationSettingsModal = ({
             )}
           </>
         )}
+
+        {/* Airtable Integration */}
+        <Typography variant="h6" sx={{ marginTop: "30px" }}>
+          {t('integration_settings.airtable.title')}
+        </Typography>
+
+        {recording && recording.airtable_base_id ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)